package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "nestjs-integration-test-db-example",
+  "version": "0.0.1",
+  "description": "This repo demonstrates how you can test a NestJS application against a real database",
+  "author": "Livio Brunner, Torsten Uhlmann",
+  "license": "UNLICENSED",
+  "scripts": {
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@lxdhub/db": "^1.10.1",
```

Comment:
We should remove that and all the lines it refers to. That was the project I have used this technique for, but it is not related to this repository ;)